conference.js

```diff
@@ -2241,19 +2245,28 @@ export default {
      * @returns {Promise}
      */
     async _onDeviceListChanged(devices) {
```

Comment:
So we no longer use this param? Should we remove it? I guess this is called after the state in 'features/base/devices' is updated, right?

Reply:
oh, ignore, we use it to filter them ... 🤦